unblob/models.py

```diff
@@ -119,9 +119,12 @@ def calculate_chunk(
         """Calculate the Chunk offsets from the Blob and the file type headers."""
 
     @staticmethod
-    @abc.abstractmethod
     def make_extract_command(inpath: str, outdir: str) -> List[str]:
-        """Make the extract command with the external tool, which can be passed for subprocess.run."""
+        """
+        Make the extract command with the external tool, which can be passed for subprocess.run.
+        Return with an empty list of the handler does not supposed to perform extraction.
```

Comment:
I would use "Returns an empty list if the handler is not supposed to perform extractions."

Reply:
Fixed, thanks!